src/PileupTrack.js

```diff
@@ -562,26 +588,54 @@ varying vec4 vColor;
                   }
 
                   let mouseOverHtml =
-                    `Position: ${read.chrName}:${
-                      read.from - read.chrOffset
-                    }<br>` +
+                    `Position: ${read.chrName}:${read.from -
+                      read.chrOffset}<br>` +
                     `Read length: ${read.to - read.from}<br>` +
-                    `MAPQ: ${read.mapq}<br>`;
+                    `MAPQ: ${read.mapq}<br>` +
+                    `Strand: ${read.strand}<br>`;
 
-                  if (nearestSub) {
-                    if (nearestSub.sub) {
-                      mouseOverHtml += `Nearest sub: ${nearestSub.type} (${nearestSub.sub.length})`;
-                    }
+                  if (nearestSub && nearestSub.type) {
+                    mouseOverHtml += `Nearest substitution: ${cigarTypeToText(nearestSub.type)} (${nearestSub.length})`;
+                  }else if(nearestSub && nearestSub.variant){
+                    mouseOverHtml += `Nearest substitution: ${nearestSub.base} &rarr; ${nearestSub.variant}`;
                   }
+
                   return mouseOverHtml;
                   // + `CIGAR: ${read.cigar || ''} MD: ${read.md || ''}`);
                 }
               }
             }
           }
         }
+
         // var val = self.yScale.domain()[index];
+        if (
+          this.options.showReadCounts &&
```

Comment:
I think what you refer to read counts is usually referred to as "coverage".

Reply:
I changed it to coverage everywhere